packages/fx-core/src/common/spec-parser/specParser.ts

```diff
@@ -72,7 +138,8 @@ export class SpecParser {
    */
   async list(): Promise<string[]> {
     try {
-      const apiMap = await this.getAllSupportedApi(this.specPath);
+      const spec = (await SwaggerParser.validate(this.specPath)) as OpenAPIV3.Document;
```

Comment:
If the path provided by the user is a remote URL, then we will do multiple GET request in scaffolding when loading operations for the user. During loading, we will call validate() first and then list(), which in total will lead to two calls. If the path parsed from OpenAI plugin manifest, then we will do three calls in total.

Reply:
If we do not want to multiple requests, we can cache the result inside the class, I can update the code to support the cache